kiali-server/templates/service.yaml

```diff
@@ -31,11 +31,21 @@ spec:
   - name: http
   {{- end }}
     protocol: TCP
+    {{- if .Values.deployment.service_port }}
+    targetPort: {{ .Values.server.port }}
+    port: {{ .Values.deployment.service_port }}
+    {{- else }}
     port: {{ .Values.server.port }}
+    {{- end }}
   {{- if .Values.server.metrics_enabled }}
   - name: http-metrics
     protocol: TCP
+    {{- if .Values.deployment.service_metrics_port }}
```

Comment:
what is "deployment.service_metrics_port"? That is not an existing setting correct? See dev checklist when adding new settings.